src/tests/SharpRaven.UnitTests/SharpRaven.UnitTests.csproj

```diff
@@ -12,34 +12,43 @@
     <ProjectReference Include="..\..\app\SharpRaven\SharpRaven.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="NSubstitute" Version="1.8.0.0" />
+    <PackageReference Include="NUnit" Version="2.6.4" />
+  </ItemGroup>
+
   <ItemGroup Condition="'$(TargetFramework)' == 'net471'">
     <Reference Include="System.Web" />
     <Reference Include="System.Configuration" />
     <Reference Include="Microsoft.CSharp" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
   </ItemGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net471'">
+    <DefineConstants>NET45PLUS_REGISTRY_VERSION;HAS_RUNTIME_INFORMATION;$(AdditionalConstants)</DefineConstants>
+  </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net35' or '$(TargetFramework)' == 'net40' or '$(TargetFramework)' == 'net45'">
```

Comment:
The references are not required on .NET 3.5 or 4.0?

Reply:
I just moved it to their own blocks as it was tricky to follow what target gets what